src/utils/object-tree-node.ts

```diff
@@ -118,11 +131,19 @@ class ObjectTreeNode implements ProxyHandler {
       if (isObject(content)) {
         changes = normalizeObject(changes, this.isObject);
         return { ...content, ...changes };
+      } else if (Array.isArray(content)) {
```

Comment:
This seems safe for ember-data.  B/c we aren't using the Ember provided `isArray`.